packages/tools/viewer-alpha/test/apps/web/index.html

```diff
@@ -88,6 +94,16 @@
             const lines = document.querySelectorAll("line");
             const hotspotResult = { screenPosition: [0, 0], worldPosition: [0, 0, 0] };
 
+            setInterval(() => {
+                // viewerElement.cameraAutoOrbit = !viewerElement.cameraAutoOrbit;
+                // viewerElement.viewerDetails.viewer.cameraAutoOrbit = !viewerElement.viewerDetails.viewer.cameraAutoOrbit;
+                // const color = viewerElement.viewerDetails.scene.clearColor.clone();
+                // color.r = Math.random();
+                // viewerElement.viewerDetails.scene.clearColor = color;
+                // console.log(viewerElement.clearColor);
+                // viewerElement.selectedAnimation = 1;
+            }, 3000);
+

```

Comment:
Did you mean to leave this in?

Reply:
Yea, maybe not the best, but this test app has a bunch of stuff commented out in various places just to make it easy to manually test different scenarios.